### ISSUE TITLE
message names not camel-cased

### ISSUE THREAD

**Issue by lestrrat** [NYTimes/openapi2proto#75]
I thought I fixed this, but 

```yaml
definitions:
   foo_bar:
      ...
```

produces a message like

```
message Foo_bar  {
...
}
```

**Comment by lestrrat** [comment]
I guess I never fixed this, as fixtures had the same problem. fixed in #77

**Comment by lestrrat** [comment]
Argh, this was not fixed in its entirety.
The definition (`message Foo { .. }`) had been fixed, but the associated return values are still problematic.

I'm going to file a fix, but at the end of the day what we probably need is a type registry of sorts, so we don't have to do this everywhere that we must refer to these types
